client-src/elements/chromedash-guide-editall-page.js

```diff
@@ -320,12 +320,16 @@ export class ChromedashGuideEditallPage extends LitElement {
     return stageIds.join(',');
   }
 
+  // render the button to add a new stage. Displays for enterprise features only.
   renderAddStageButton() {
-    const text = this.feature.is_enterprise_feature ? 'Add Step': 'Add Stage';
+    if (!this.feature.is_enterprise_feature) {
```

Comment:
Maybe we can user renderHTMLif from utils

Reply:
Good idea - changed